tests/lib/ImageThumbnailTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+use PHPUnit\Framework\TestCase;
+use Mre\Unicorn\lib\Image;
+
+/**
+ * Image Thumbnail test class
+ */
+
+class ImageThumbnailTest extends TestCase
+{
+  public function testImageMustBeOfValidExtension() {
+    $imageFile = Image::createThumbnail('//img.url/myAvatar.pngh');
```

Comment:
`Image::createThumbnail` is not returning any values

Reply:
I also noticed the bug, however, I felt maybe it was the data I was passing that's why it throwing the error. I didn't want to break what's already working that's why I didn't touch it.
`Image::createThumbnail` throws an error on line `50` in `lib\Image.php` file.

Once this is fixed the test will pass.

Reply:
You are right it is a bug. We get an exception when we get file that has non image extension (like jpeg, jpg, png, gif). We will try to fix in a different PR.

But your test is not passing because of a different problem. 
1. You are providing URL `//img.url/myAvatar.pngh` with wrong `.pngh` extension.
2. URL is pointing to invalid image. I mean, there is not actual image at `//img.url/myAvatar.pngh` so it crashes. I suggest adding local image to use it in tests.

Reply:
@waleCloud, thanks for your PR. We'd love to merge this. Seems like there is just this little piece missing here to get it done. Do you have time to get it done or shall we support? 😊

Reply:
@mre Thanks for heads up! 
I have updated the code to support an actual image file, but that will probably not solve the issue as the `Image` class in the **lib** dir currently breaks at line 
> 50, 51,  causing lines 53, 63,79 & 81 to also fail.

Once lines `50 & 51` are fixed, everything should be fixed.
However, I will appreciate any support here 😃.
Thanks